### PR TITLE
HBW-77 Show alert if form with file upload field has no file list field

### PR DESCRIPTION
**Pull request by TimAle**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,7 +25,6 @@
     "react_ujs": "^2.4.4",
     "select2": "^4.0.6-rc.1",
     "tooltip.js": "^1.3.0",
-    "webpack-merge": "^4.1.4",
```
Comment (TimAle) [inline package.json]: It is unused, I forgot to remove it in #171

File: hbw/config/locales/en.yml [hunk 2]
```diff
@@ -15,6 +15,7 @@ en:
         field_not_defined_in_bp: "Field with name %{field_name} not defined in BP variables"
         cannot_obtain_available_actions: Cannot obtain available actions
         cannot_start_process: Cannot start process
+        file_list_field_required: File list field with name "homsOrderDataFileList" required
```
Comment (sugarfree1) [inline hbw/config/locales/en.yml]: You write about type in Russian locale, but don't write about it here.

**Comment by TimAle** [comment]
UPD

**Comment by TimAle** [comment]
UPD

**Pull request merged**